### ISSUE TITLE
Incompatible Swift version - framework was built with 5.1.3 (swiftlang-1100.0.282.1 clang-1100.0.33.15) and the local version is 5.2 (swiftlang-1103.0.32.1 clang-1103.0.32.29).

### ISSUE THREAD

**Issue by walkertop** [mozilla-mobile/firefox-ios#6314]
Describe the bug

<img width="1636" alt="image" src="https://user-images.githubusercontent.com/17690332/77648676-80d4bf00-6fa3-11ea-97c9-b532c83cce14.png">
***  Skipped installing application-services.framework binary due to the error:
	"Incompatible Swift version - framework was built with 5.1.3 (swiftlang-1100.0.282.1 clang-1100.0.33.15) and the local version is 5.2 (swiftlang-1103.0.32.1 clang-1103.0.32.29)."

    Falling back to building from the source
Specifications

OSX 10.15.4
xcode 11.4
carthage 0.34.0

suggestion
maybe it needs to update some relevant framework for swift version 5.2, such as application-services

**Comment by armcknight** [comment]
Thanks for filing this, just came here to do the same. 

I spent some time yesterday trying to build `application-services` from source (https://github.com/mozilla/application-services/issues/2872), which I finally did get working, but that still results in a build error in firefox being unable to find the `MozillaAppServices` module. I confirmed there's no MozillaAppServices.framework in Carthage/Build, and not sure how to generate it.
